SU-SWS/decanter
# ADAPT-1234: Did thing.

# NOT READY

# Summary
- Adds Matt to changelog
- Fixes bug 1,2 and 3.

# Needed By (Date)
- EOD Monday 9th

# Urgency
- Meh.

# Steps to Test

1. Check out branch 234 of X project
2. Npm install the new packages
3. Run netlify dev
4. Go to page /xyz
5. Validate classname thing on element 7

# Affected Projects or Products
- Drupal project
- Adils thing

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
